fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/IcebergTransaction.java

```diff
@@ -59,140 +66,114 @@ public void updateIcebergCommitData(List<TIcebergCommitData> commitDataList) {
         }
     }
 
-    public void beginInsert(String dbName, String tbName) {
-        Table icebergTable = ops.getCatalog().loadTable(TableIdentifier.of(dbName, tbName));
-        transaction = icebergTable.newTransaction();
+    public void pendingCommit(SimpleTableInfo tableInfo) {
```

Comment:
If possible, we try to keep all table insertions using the same function name, including hive, iceberg, and paimon, hudi, etc. that may be supported later.

Reply:
This can be unified with hive, but I think pendingCommit, precommit and commit are more suitable, and hive can also move towards this aspect.